kolibri/logger/serializers.py

```diff
@@ -101,7 +102,7 @@ def get_currentmasterylog(self, obj):
         except MasteryLog.DoesNotExist:
             return None
 
-class UserSessionLogSerializer(serializers.ModelSerializer):
+class UserSessionLogSerializer(KolibriModelSerializer):
```

Comment:
Reading the REST framework docs, this really is the only pattern, right? There's no way of specifying a custom field type's serialization on the Field class, right? And no settings? I couldn't find any... just kind of surprised that this is the way to do it :)